pkg/resources/container.go

```diff
@@ -27,6 +30,23 @@ func NewInstanceContainer(instance *kvrocksv1alpha1.KVRocks) *corev1.Container {
 	return container
 }
 
+func NewExporterContainer(instance *kvrocksv1alpha1.KVRocks) *corev1.Container {
+	return &corev1.Container{
+		Name:  "kvrocks-exporter",
+		Image: "jinxu95/kvrocks_exporter:latest",
```

Comment:
Hi @ColinChamber .
This image needs to be replaced with the official.

Reply:
Please review https://github.com/RocksLabs/kvrocks_exporter/pull/23